libbeat/outputs/fileout/docs/fileout.asciidoc

```diff
@@ -49,6 +49,14 @@ The default value is `true`.
 The path to the directory where the generated files will be saved. This option is
 mandatory.
 
+The path may include the timestamp when the process is started using the `+FORMAT` syntax where `FORMAT` is a
+valid https://godoc.org/github.com/elastic/beats/libbeat/common/dtfmt[time format],
```

Comment:
This link doesn't seem to work b/c of our dual license for the project.

Reply:
Maybe we could link to the source file in GH until we can put that FORMAT content into the docs. I think there is an acsiidoc variable for the branch name so you could link to the relevant branch.

https://github.com/elastic/beats/blob/997757b7b92faf14e8fa2345ae69d9dbb225e141/libbeat/docs/version.asciidoc#L2

Reply:
updated with a reference to that variable. Hopefully I did it correctly.